src/components/SearchPerson/SearchPerson.tsx

```diff
@@ -1,16 +1,184 @@
 import * as React from 'react'
-//import { gql } from '@apollo/client'
-import { Row, Col } from 'react-bootstrap'
+import { Row, Alert } from 'react-bootstrap'
+import { gql, useQuery } from '@apollo/client'
+import { useQueryState } from 'next-usequerystate'
+// eslint-disable-next-line no-unused-vars
+import {  PersonType } from "../PersonContainer/PersonContainer"
+import Pager from "../Pager/Pager"
+import ContentLoader from "react-content-loader"
+import Error from "../Error/Error"
+import PersonMetadata from "../PersonMetadata/PersonMetadata"
+
 
 type Props = {
     searchQuery: string;
 };
 
+
+interface ContentQueryData {
+    people: {
+      __typename: String
+      nodes: PersonType[]
+      pageInfo: PageInfo
+    },
+  }
+
+interface PageInfo {
+    endCursor: string
+    hasNextPage: boolean
+}
+  
+interface ContentQueryVar {
+    query: string
+    cursor: string
+}
+
+export const CONTENT_GQL = gql`
+  query getContentQuery($query: String, $cursor: String) {
+    people(first: 25, query: $query, after: $cursor) {
+    nodes {
+      id
+      name
+      givenName
+      familyName
+      # citationCount
+      # viewCount
+      # downloadCount
+      affiliation {
+        name
+        id
+      }
+      works{
+        totalCount
+      }
+    }
+  }
+}
+`
+
 const SearchPerson: React.FunctionComponent<Props> = ({ searchQuery }) => {
+    // const router = useRouter()
+
+    /* eslint-disable no-unused-vars */
+    const [cursor, setCursor] = useQueryState('cursor', { history: 'push' })
+    /* eslint-enable no-unused-vars */
+    const [searchResults, setSearchResults] = React.useState([])
+    const { loading, error, data, refetch } = useQuery<ContentQueryData, ContentQueryVar>(
+      CONTENT_GQL,
+      {
+        errorPolicy: 'all',
+        variables: { query: searchQuery, cursor: cursor }
+      }
+    )
+
+    React.useEffect(() => {
+        const typingDelay = setTimeout(() => {
```

Comment:
This doesn't need to exist here now as the query should always come in from the prop?